### PR TITLE
Update SVGAElement to match attributes on HTMLAnchorElement

### PR DESCRIPTION
**Pull request by dstorey**
* Harmonised existing IDL attributes with the definitions in HTML (inc. setting Animatable to false and no longer read only)
* Added IDL attributes missing from SVG but included in HTML - ping, referrerPolicy and text. These 3 are in the process of being implemented in Firefox.

This is temporary until the HTML spec includes the HTMLOrSVGAnchorElement mixin (or equivalent), when we can just refer to that spec. I just copied the reflect rules for `text` from HTML for example. 

Apologies if I didn't add things in all the right places or got the references wrong. I'm still learning how things should be structured and edited.

### CODE REVIEW

**Comment by dstorey** [comment]
Tests for some of these: https://github.com/w3c/web-platform-tests/pull/10275
Patch for Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1451823

File: master/definitions.xml [hunk 1]
```diff
@@ -16,11 +16,13 @@
       interfaces='SVGAElement'>
     <x:contentmodel xmlns='http://www.w3.org/1999/xhtml'>Descriptive content, plus any element or text allowed by its parent's content model, except for another a element.  If the parent is a switch element, use the content model of the nearest ancestor that isn't a switch.</x:contentmodel>
     <attribute name='href' href='linking.html#AElementHrefAttribute' animatable='yes'/>
-    <attribute name='target' href='linking.html#AElementTargetAttribute' animatable='yes'/>
-    <attribute name='download' href='linking.html#AElementDownloadAttribute' animatable='yes'/>
-    <attribute name='rel' href='linking.html#AElementRelAttribute' animatable='yes'/>
-    <attribute name='hreflang' href='linking.html#AElementHreflangAttribute' animatable='yes'/>
-    <attribute name='type' href='linking.html#AElementTypeAttribute' animatable='yes'/>
+    <attribute name='target' href='linking.html#AElementTargetAttribute' animatable='no'/>
```
Comment (dirkschulze) [inline master/definitions.xml]: `target` is animatable in WebKit as far as I can see. This was implemented before the Blink split so I am positive that it is in Blink as well. Could you add the link to the WG resolution if this was resolved on please?

Comment (ewilligers) [reply master/definitions.xml]: target is an SVGAnimatedString in SVG 1.1 and in all browsers including Edge.

The following example works in all browsers, using target.baseVal to read the target.
https://jsfiddle.net/ericwilligers/bL9zvd1w/

Comment (dstorey) [reply master/definitions.xml]: we agreed to change target to DOMString as the usage is so tiny.

Comment (AmeliaBR) [reply master/definitions.xml]: Here's the resolution for that decision:
https://github.com/w3c/svgwg/issues/315#issuecomment-376271318

**Comment by AmeliaBR** [comment]
Okay, I'm merging in the changes to everything except `target`, as discussed on the call today (see https://github.com/w3c/svgwg/pull/409#issuecomment-412654080).

This means the spec matches the Firefox implementation.  And hopefully means that other browsers can move ahead with matching _their_ implementations!

We'll continue discussion of the potential breaking change re `target` in a separate issue (which I'll link here once it exists).

**Pull request merged**